api/Console.json

```diff
@@ -0,0 +1,1048 @@
+{
+  "api": {
+    "Console": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Console",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": "4"
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "2"
+          },
+          "firefox_android": {
+            "version_added": true
+          },
+          "ie": {
+            "version_added": "8",
+            "partial_implementation": true,
+            "notes": [
+              "Only functions when developer tools are opened. Otherwise, the 'console' object is undefined."
+            ]
+          },
+          "opera": {
+            "version_added": "10.1"
+          },
+          "safari": {
+            "version_added": "3.1"
+          },
+          "safari_ios": {
+            "version_added": "3.2"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "assert": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/console/assert",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "28",
+              "notes": [
+                "Available in workers from 38 onwards"
```

Comment:
Lets please remove the notes throughout the file and add worker support as a sub feature of console like so:

```json
"worker_support": {
  "__compat": {
    "description": "Available in workers",
    "support": {
```
In the tree this would get added here:
- api
  - Console
    - __compat
    - **worker_support**
    - assert
    - ...